.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -621,13 +621,12 @@ stages:
         patterns: '**/*-$(targetPlatform).msi'
         path: $(System.DefaultWorkingDirectory)/$(relativeMsiOutputDirectory)
 
-    - script: tracer\build.cmd BuildWindowsIntegrationTests PublishIisSamples -Framework $(framework)
-      displayName: BuildWindowsIntegrationTests
-
     - script: |
-        docker-compose build --build-arg dotnet_tracer_msi=.$(relativeMsiOutputDirectory)/*.msi --build-arg ENABLE_32_BIT=$(enable32bit) IntegrationTests.IIS
+        tracer\build.cmd BuildWindowsIntegrationTests PublishIisSamples -Framework $(framework) &
+        docker-compose build --build-arg dotnet_tracer_msi=.$(relativeMsiOutputDirectory)/*.msi --build-arg ENABLE_32_BIT=$(enable32bit) IntegrationTests.IIS &
```

Comment:
This risks being flaky - the docker-compose build depends on the output of the `PublishIisSamples` step, so we _need_ them to be sequential, not parallelised

Reply:
I missed the fact that we depended on this one sample. I'm even puzzled it worked actually cause this part is way faster than the build so I would have guessed, it would be not only flaky, but constantly failing.

Reply:
Fixed in 3c899690453315bbb2a45dbbd6e33fa05061aa89

Reply:
> I missed the fact that we depended on this one sample. I'm even puzzled it worked actually cause this part is way faster than the build so I would have guessed, it would be not only flaky, but constantly failing.

Seems like a red flag 😬  maybe we should test removing the PublishIisSamples stage entirely and confirm it _does_ fail!